internal/action/verify.go

```diff
@@ -126,6 +126,11 @@ func VerifyReleaseObject(snapshot *v2.Snapshot, rls *helmrelease.Release) error
 	verifier := relDig.Verifier()
 
 	obs := release.ObserveRelease(rls)
+
+	// unfortunately we have to pass in the OciDigest as is, because helmrelease.Release
+	// does not have a field for it.
+	obs.OciDigest = snapshot.OciDigest
```

Comment:
I cannot think of another way yet... The `oci digest` must be part of the `snapshot` object in order to be consistent.